x-pack/plugins/security_solution/common/detection_engine/schemas/request/rule_schemas.ts

```diff
@@ -159,12 +159,11 @@ const commonParams = {
     tags,
     interval,
     enabled,
-    throttle: throttleOrNull,
+    throttle,
```

Comment:
Minor API change here, no longer allowing `null` to be sent in as a value for throttle. We already use `throttle: 'no_actions'` in the UI and API tests to represent the same concept that `null` represents here, and `null` is not listed as an allowed value in the [API docs](https://www.elastic.co/guide/en/security/current/rules-api-create.html).

The driving reason for making this change is that the existing [response schema](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/detection_engine/schemas/response/rules_schema.ts#L167) defines `throttle` as a string, and this newer schema builds both the request and response schema using the same structure to ensure parity. So the best way to get consistency between the existing response and the new schema is to simply disallow `null` as an input in this schema.